apps/src/sites/studio/pages/feedback/index.js

```diff
@@ -0,0 +1,23 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+$(document).ready(showFeedback);
+
+function showFeedback() {
+  const script = document.querySelector('script[data-feedback]');
+  const feedbackData = JSON.parse(script.dataset.feedback);
+
+  ReactDOM.render(
+    <div>
```

Comment:
since we know this (lines 12-20) will become a React component, maybe we should set that up now?